src/data-server/sbvr-server.ts

```diff
@@ -411,4 +375,38 @@ export async function setup(app, sbvrUtils, db) {
 	});
 
 	await db.transaction(setupModels);
-}
+};
+
+export const config: Config = {
```

Comment:
I just had to change the order config & setup variabels were defined as config uses the "setup" function and TS would complain we were defining it after having used it

Reply:
That was because you changed it from being a function declaration which would be hoisted, into a function assignment to variable. Which was so it was easier to do the `SetupFunction` typing which is fine (I think it's technically possible to do with a function declaration.. but it's such a pain I also always avoid it)